src/Super_Admin_Command.php

```diff
@@ -23,11 +23,11 @@
 class Super_Admin_Command extends WP_CLI_Command {
 
 	private $fields = array(
-		'user_login'
+		'user_login',
 	);
 
 	public function __construct() {
-		$this->fetcher = new \WP_CLI\Fetchers\User;
+		$this->fetcher = new \WP_CLI\Fetchers\User();
```

Comment:
While we're make CS changes, let's import classes like these so we can see the dependencies at the top of the file and have clean class names in the code.

As the pure class name here is not very telling without the preceding namespace (`User`), I'd add an alias to the import at the top:
```php
use WP_CLI\Fetchers\User as UserFetcher;
```
Then, the code becomes clear and nice here:
```suggestion
		$this->fetcher = new UserFetcher();
```